packages/contracts-bedrock/interfaces/dispute/IDelayedWETH.sol

```diff
@@ -3,15 +3,9 @@ pragma solidity ^0.8.0;
 
 import { ISystemConfig } from "interfaces/L1/ISystemConfig.sol";
 import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
-import { IProxyAdmin } from "interfaces/universal/IProxyAdmin.sol";
-
-interface IDelayedWETH {
```

Comment:
@smartcontracts is there a reason we had been avoiding using inheritance in interfaces?

Reply:
Seems fine, sometimes the inheritance gets wonky in the compiler but I'm ok with this if it compiles and passes CI